src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -326,7 +326,7 @@ namespace PlantLoopEquip {
 				}
 
 			} else if ( EquipTypeNum == TypeOf_Chiller_Absorption ) {
-				SimBLASTAbsorber( sim_component.TypeOf, sim_component.Name, EquipFlowCtrl, LoopNum, LoopSideNum, EquipNum, RunFlag, FirstHVACIteration, InitLoopEquip, CurLoad, MaxLoad, MinLoad, OptLoad, GetCompSizFac, SizingFac, TempCondInDesign );
```

Comment:
@mbadams5 @Myoldmopar @EnergyArchmage .I'm not 100% certain, but I'm assuming .TypeOf is there as an argument to facilitate error messages.  Even though it's not presently being used, it's another case of maintaining some symmetry.